diff -urNp -x '*.orig' camlidl-camlidl109.org/lib/Makefile camlidl-camlidl109/lib/Makefile
--- camlidl-camlidl109.org/lib/Makefile	2020-05-22 16:45:29.000000000 +0200
+++ camlidl-camlidl109/lib/Makefile	2021-03-23 21:48:44.434965400 +0100
@@ -28,8 +28,13 @@ $(BYTELIB): $(BYTEOBJS)
 $(NATIVELIB): $(NATIVEOBJS)
 	$(OCAMLOPT) -a -o $(NATIVELIB) -cclib -lcamlidl $(NATIVEOBJS)
 
-install:
-	cp $(INTERFACES) $(BYTELIB) $(NATIVELIB) $(NATIVELIB:.cmxa=.$(LIBEXT)) $(OCAMLLIB)
+install: install-byte $(if $(NATIVELIB),install-opt)
+
+install-byte:
+	cp $(INTERFACES) $(BYTELIB) $(OCAMLLIB)
+
+install-opt:
+	cp $(NATIVELIB) $(NATIVELIB:.cmxa=.$(LIBEXT)) $(OCAMLLIB)
 	cd $(OCAMLLIB); $(RANLIB) $(NATIVELIB:.cmxa=.$(LIBEXT))
 
 .SUFFIXES: .mli .ml .cmi .cmo .cmx
